docs/ops/condition/Select_1.md

```diff
@@ -17,26 +17,31 @@
 
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
-    * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
-  * **Type**: string
+    * *none* - no auto-broadcasting is allowed, all input shapes must match,
```

Comment:
Maybe remove the commas at the end of the description in ranges?

Reply:
Done in this [commit](https://github.com/openvinotoolkit/openvino/pull/6488/commits/22ec1107bcfa18991452f79637c5215ac9d94298).